XRTK.Lumin/Packages/com.xrtk.lumin/Controllers/LuminController.cs

```diff
@@ -61,7 +61,7 @@ public override void SetupDefaultInteractions(Handedness controllerHandedness)
         /// <summary>
         /// Updates the controller's interaction mappings and ready the current input values.
         /// </summary>
-        public void UpdateController()
+        public override void UpdateController()
```

Comment:
Might revert this part, it was in line with trying to standardise the UpdateController function across all platforms